### ISSUE TITLE
X20HD support

### ISSUE THREAD

**Issue by kakashi2018** [FrSkyRC/ETHOS-Feedback-Community#1970]
Can we expect firmware support for x20hd version in 1.4? We have already got the radios and just curios to know that I can use it with the ELRS support from 1.4.

**Comment by Linseder** [comment]
That is not correct.
I've had my X20HD upkit since mid-August. Bought from Engel Modellbau. Installed with Ethos 1.2.12

**Comment by kakashi2018** [comment]
@bsongis-frsky yes, I have mine coming up next Monday. Can we have an estimate date on when we will have the firmware please for the elrs support?

**Comment by bsongis-frsky** [comment]
We will release 1.4.1 soon with minor fixes for all radios, except X20-HD
Then I will work on X20-HD
